src/main/java/org/opentripplanner/routing/core/vehicle_sharing/KickScooterDescription.java

```diff
@@ -70,9 +70,7 @@ public boolean equals(Object o) {
         if (this == o) return true;
         if (o == null || getClass() != o.getClass()) return false;
         KickScooterDescription that = (KickScooterDescription) o;
-        return Double.compare(getLongitude(), that.getLongitude()) == 0 && // TODO AdamWiktor VMP-4 compare approximately
-                Double.compare(getLatitude(), that.getLatitude()) == 0 &&
-                Double.compare(getRangeInMeters(), that.getRangeInMeters()) == 0 &&
+        return Objects.equals(getProviderVehicleId(), that.getProviderVehicleId()) &&
```

Comment:
Hulajnogi Lime'a będą wszystkie inne ale to jest ok.

Reply:
Aktualnie nie mamy żadnych hulajnóg od `Lime` w bazie danych, jak się pojawią, to się będziemy martwić